search bar redux

Elodie-Gr opened the pull request.

Elodie-Gr committed:
search bar redux

Elodie-Gr committed:
search bar difficulty redux

Marksu-u commented on YourFitnessBuddy/src/components/filterDifficulty/index.js:
```diff
@@ -0,0 +1,43 @@
+        Choose your difficulty !
+      </Text>
+      <SearchContainer>
+      <SearchInput placeholder="Beginner, Intermediate or Expert" value={query} onChangeText={handleInputChange} />
```
C'est possible d'avoir une liste étant donné que les difficultés sont communes à tous les exercices ?

Marksu-u replied:
Regardes si tu peux faire ça derrière, je vais voir si on peut rassembler les deux filtres en un seul appel

Marksu-u committed:
rewrite code

- removed redux actions/reducers for filter
- merge the filter action into one route
- renamed routes
- typo fix

Marksu-u committed:
fix warn & removed unused files

- onPress -> onValueChange

Marksu-u committed:
refonte front
